lib/fluentReports.js

```diff
@@ -3426,25 +3426,33 @@
             if (options.dash) { this._PDF.undash(); }
         },
 
-        _truncateText: function (data, width) {
-            var curData;
-            var offset = data.data.indexOf('\n');
-            if (offset > 0) {
-                curData = data.data.substr(0, offset);
-            } else {
-                curData = data.data;
-            }
-            var wos = this.widthOfString(curData);
+        _truncateText: function (data, width, lineData) {
+            width -= this.widthOfString('M'); // Due to rendering differences, purposefully subtract a wide character from the width. (1 em)
+            var string = data.data,
+                firstNewline = data.data.indexOf('\n');
+            if (firstNewline !== -1) {
+                string = data.data.substr(0, firstNewline);
```

Comment:
Double assignment of the "string" variable is a bad thing when dealing with loops that may call this thousands to millions of times.      The assignment should occur either in the `if` or `else` side; not in init a (line 3431) and then `if`, because then you have two assignments when the `if` is true...